mmv1/templates/terraform/examples/secure_source_manager_branch_rule_with_fields.tf.erb

```diff
@@ -0,0 +1,33 @@
+resource "google_secure_source_manager_instance" "instance" {
+    location = "us-central1"
+    instance_id = "<%= ctx[:vars]['instance_id'] %>"
+    # Prevent accidental deletions.
+    lifecycle {
+        prevent_destroy = "<%= ctx[:vars]['prevent_destroy'] %>"
+    }
+}
+
+resource "google_secure_source_manager_repository" "repository" {
+    repository_id = "<%= ctx[:vars]['repository_id'] %>"
+    instance = google_secure_source_manager_instance.instance.name
+    location = google_secure_source_manager_instance.instance.location
+    # Prevent accidental deletions.
+    lifecycle {
+        prevent_destroy = "<%= ctx[:vars]['prevent_destroy'] %>"
+    }
+}
+
+resource "google_secure_source_manager_branch_rule" "default" {
+    branch_rule_id = "<%= ctx[:vars]['branch_rule_id'] %>"
+    location = google_secure_source_manager_repository.repository.location
+    repository_id = google_secure_source_manager_repository.repository.repository_id
+    include_pattern = "test"
+    minimum_approvals_count   = 2
+    minimum_reviews_count     = 2
+    require_comments_resolved = true
+    require_linear_history    = true
+    require_pull_request      = true
+    disabled = false
+    allow_stale_reviews = false
+    etag = "default"
```

Comment:
I don't think we use the etag field, could probably just leave it blank
@umairidris

Reply:
That's true, I could just remove it from the configuration.